apps/settings/css/settings.scss

```
@@ -137,7 +137,7 @@ select {
 	padding: 14px 20px;
```

Padding needs than to be adjusted, otherwise this looks weird.

```suggestion
	padding: calc(var(--default-grid-baseline) * 2) calc(var(--default-grid-baseline) * 3);
```